package.json

```diff
@@ -45,9 +46,11 @@
   "devDependencies": {
     "@magento/eslint-config": "~1.3.0",
     "babel-eslint": "~10.0.1",
+    "babel-plugin-module-resolver": "^3.1.3",
```

Comment:
Consider `--tilde` here.

Reply:
Whoa, how'd that get in there?

Reply:
Looks like Yarn doesn't respect our .npmrc, so I added a .yarnrc to set the default save prefix.